bin/cli.js

```diff
@@ -48,7 +47,6 @@ const cliFlags = yargs
   })
   .option('config', {
     'describe': 'Path to json config file',
-    'coerce': (arg) => getConfigFromFile(arg),
```

Comment:
If we remove this function then we should modify [this line](https://github.com/paulirish/pwmetrics/blob/6f2647fa22c21181e1669ae6e467a845971e94dd/bin/cli.js#L60)  accordingly. When the ```check```function executes it will always (I think) have ```argv.config``` set to empty string which will create false positives.

I'm curious, could you explain a little about your thoughts on this particular line?